starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,87 @@
+*{
+
+margin: 0;
+padding: 0;
+
+}
+
+body{
+line-height: 1.6;
+font-family: ‘Montserrat’, sans-serif;
+
+}
+
+
+h2 {
+    padding-bottom: 20px;
+}
+
+.cover {
+    background-image: url(../images/apple-pie.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 1100px;
+    width: 100%;
+    margin-bottom: 50px;
+    text-align: center;
+    color: white;
+
+}
+
+.cover h1{
+
+position: absolute;
+top: 50%;
+left: 40%;
+font-size: 5em;
+
+}
+
+
+.applepie-background img{
+width: 100%;
+height: 100%;
+}
+
+
+.description{
+    text-align: center;
+}
+.ingredients{
+    
+    text-align: left;
+    margin: 0 10%;   
+}
+
+.ingredients li {
+    list-style: '-' ;
```

Comment:
Good call to use a ul with lis for this part, and to try to change the list style type in the css!
since '-' is not a css built-in list style type, the syntaxe to do that however is a bit more complicated and it won't work by just hardcoding it like that.  This is the correct syntax to do that, but you will learn about that later in the course : 
.ingredients li::before{
    content:"- ";
  }